client/src/main/java/com/tigrisdata/db/client/model/ReadFields.java

```diff
@@ -0,0 +1,55 @@
+package com.tigrisdata.db.client.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.tigrisdata.db.client.utils.Utilities;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public final class ReadFields {
+  private final Map<String, Object> internalMap;
+  private static final ReadFields EMPTY = new ReadFields(Collections.emptyMap());
```

Comment:
empty means everything returned from the server even if fields is not in JSON.

Reply:
Client will now not send this field to server if it is empty.